.gitignore

```diff
@@ -14,3 +14,4 @@ tmp
 build/
 generated_code/
 node_modules/
+*-packr.go
```

Comment:
Is this autogenerated code being used by the applicaiton? If so I think we should push it as well

Reply:
It is not recommended by packr. In fact I changed the way to build to allow troubleshooting the process locally (review makefile)